HDF5Examples/C/H5FLT/h5ex_d_granularbr.c

```diff
@@ -0,0 +1,238 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of the HDF5 Granular BitRound filter plugin source.  The full    *
+ * copyright notice, including terms governing use, modification, and        *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the file COPYING, which can be found at the root of the Granular BitRound source code   *
+ * distribution tree.  If you do not have access to this file, you may       *
```

Comment:
BTW, where is this COPYING file?  In https://github.com/HDFGroup/hdf5_plugins/tree/master/BITROUND?

Reply:
Cut-n-paste from pluguns folder - will fix